python/benchmarks/sift.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
```

Comment:
Is this confusing that we have two different benchmark directories?

Reply:
Oh, right. lemme merge this to root benchmarks.